<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>equipmentnews</class>
 <widget class="QDialog" name="equipmentnews">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>936</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(211,211,211);
background-position: center center;</string>
  </property>
  <widget class="QPushButton" name="btn_return">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>0</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#00FFFF;
	border-style:outset;
	border-radius:10px;
</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_look">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>290</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#00FFFF;

	border-style:outset;
	border-radius:10px;
</string>
   </property>
   <property name="text">
    <string>查看所有装备信息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_del">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>90</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#00FFFF;

	border-style:outset;
	border-radius:10px;
</string>
   </property>
   <property name="text">
    <string>删除装备信息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_update">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>190</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#00FFFF;
	border-style:outset;
	border-radius:10px;
</string>
   </property>
   <property name="text">
    <string>修改装备信息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_XML">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>390</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#00FFFF;
	border-style:outset;
	border-radius:10px;
</string>
   </property>
   <property name="text">
    <string>XML</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_EXCEL">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>490</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#00FFFF;
	border-style:outset;
	border-radius:10px;
</string>
   </property>
   <property name="text">
    <string>EXCEL</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_name">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>50</y>
     <width>237</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #000000;
background-color: rgb(255, 255, 255);
	font-size:18px;
	border-style:outset;
	border-radius:10px;</string>
   </property>
   <property name="placeholderText">
    <string>装备名称</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_classify">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>236</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #000000;
background-color: rgb(255, 255, 255);
	font-size:18px;
	border-style:outset;
	border-radius:10px;</string>
   </property>
   <property name="placeholderText">
    <string>类型</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_quality">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>237</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #000000;
background-color: rgb(255, 255, 255);
	font-size:18px;
	border-style:outset;
	border-radius:10px;</string>
   </property>
   <property name="placeholderText">
    <string>品质</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_class">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>100</y>
     <width>236</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #000000;
background-color: rgb(255, 255, 255);
	font-size:18px;
	border-style:outset;
	border-radius:10px;</string>
   </property>
   <property name="placeholderText">
    <string>等级</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_speed">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>190</y>
     <width>236</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #000000;
background-color: rgb(255, 255, 255);
	font-size:18px;
	border-style:outset;
	border-radius:10px;</string>
   </property>
   <property name="placeholderText">
    <string>攻击速度</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>330</y>
     <width>61</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>附加属性</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ok">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>530</y>
     <width>286</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#00FFFF;
	border-style:outset;
	border-radius:10px;</string>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>427</x>
     <y>350</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>其他信息</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_otherInfomation">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>220</y>
     <width>237</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #000000;
background-color: rgb(255, 255, 255);
	font-size:18px;
	border-style:outset;
	border-radius:10px;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_additiveAttribute">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>237</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #000000;
background-color: rgb(255, 255, 255);
	font-size:18px;
	border-style:outset;
	border-radius:10px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_atk">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>140</y>
     <width>237</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #000000;
background-color: rgb(255, 255, 255);
	font-size:18px;
	border-style:outset;
	border-radius:10px;</string>
   </property>
   <property name="placeholderText">
    <string>攻击力</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 16pt &quot;Microsoft YaHei UI&quot;;
background:rgba(85,170,225,0);
	font-style: italic bold;
	font size:14px;</string>
   </property>
   <property name="text">
    <string>装备信息</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ID">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>237</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #000000;
background-color: rgb(255, 255, 255);
	font-size:18px;
	border-style:outset;
	border-radius:10px;</string>
   </property>
   <property name="placeholderText">
    <string>ID</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_XML2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>530</y>
     <width>286</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#00FFFF;
	border-style:outset;
	border-radius:10px;</string>
   </property>
   <property name="text">
    <string>导入</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
